src/app.js

```diff
@@ -1,5 +1,206 @@
 const express = require('express');
 
+const { sayHello } = require('./lib/strings');
+const { firstCharacters } = require('./lib/strings');
+const { uppercase } = require('./lib/strings');
+const { lowercase } = require('./lib/strings');
+const { firstCharacter } = require('./lib/strings');
+
+const { add, subtract, divide, multiply, remainder } = require('./lib/numbers');
+const { negate, truthiness, isOdd, startsWith } = require('./lib/booleans');
+const {
+  getNthElement,
+  arrayToCSVString,
+  addToArray2,
+  elementsStartingWithAVowel,
+  removeNthElement2,
+} = require('./lib/arrays');
+
+// Middleware
+
 const app = express();
+app.use(express.json());
+
+// Strings Section
+
+app.get('/strings/hello/world', (_, res) => {
+  res.status(200).json({ result: 'Hello, world!' });
+});
+
+app.get('/strings/hello/:string', (req, res) => {
+  res.status(200).json({ result: sayHello(req.params.string) });
+});
+
+app.get('/strings/upper/:string', (req, res) => {
+  // const result = req.params.string.toUpperCase();
+  res.status(200).json({ result: uppercase(req.params.string) });
+});
+
+app.get('/strings/lower/:string', (req, res) => {
+  res.status(200).json({ result: lowercase(req.params.string) });
+});
+
+app.get('/strings/first-character/:string', (req, res) => {
+  res.status(200).json({ result: firstCharacter(req.params.string) });
+});
+
+app.get('/strings/first-characters/:string', (req, res) => {
+  const n = req.query.length;
+  res.status(200).json({ result: firstCharacters(req.params.string, n) });
```

Comment:
No need to create a variable here, just reference the value direct in the return statement:
```
  res.status(200).json({ result: firstCharacters(req.params.string, req.query.length) });
```